jenkins-jnlp-slave/Dockerfile

```diff
@@ -0,0 +1,29 @@
+ARG SOURCE_CONANIO_IMAGE
+
+#Usage: docker build <docker image name> --build-arg SOURCE_CONANIO_IMAGE=<conanio dockerhub image> .
+FROM $SOURCE_CONANIO_IMAGE
+
+USER root
+
+ARG AGENT_VERSION=3.27
+
+#Install java to launch slave.jar
```

Comment:
Is it possible to copy jenkins file before to run these steps? you could unify all step to be executed in a single RUN.

Reply:
ok. 👍